jupyter-notebooks/analysis-ready-data/ard_1_intro_and_best_practices.ipynb

```diff
@@ -892,27 +967,38 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 33,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "def show_ndvi(ndvi):\n",
-    "    fig = plt.figure(figsize=(20,10))\n",
+    "    \"\"\"\n",
+    "    Displays the NDVI (Normalized Difference Vegetation Index) \n",
+    "    image using a diverging color scheme.\n",
+    "\n",
+    "    Args:\n",
+    "        ndvi (numpy.ndarray): A NumPy array representing the NDVI image.\n",
+    "\n",
+    "    Returns:\n",
+    "        None. Displays NDVI image.\n",
+    "    \"\"\"\n",
+    "    fig = plt.figure(figsize=(20, 10))\n",
     "    ax = fig.add_subplot(111)\n",
     "\n",
-    "    # diverging color scheme chosen from https://matplotlib.org/users/colormaps.html\n",
-    "    cmap = plt.cm.RdYlGn \n",
+    "    # Diverging color scheme chosen from https://matplotlib.org/users/colormaps.html\n",
+    "    cmap = plt.cm.RdYlGn\n",
     "\n",
     "    mmin = np.nanmin(ndvi)\n",
     "    mmax = np.nanmax(ndvi)\n",
     "\n",
     "    mid = 0\n",
     "\n",
     "    cax = ax.imshow(ndvi, cmap=cmap, clim=(mmin, mmax),\n",
```

Comment:
I'm not sure how much we want to do with figures in this PR, but at the very least, can we please make the `mmin` and `mmax` for each the same? I'd recommend -1 to +1.

Reply:
we have spoken about this, leaving it and fixing in another PR